Task5/Valentin Dimitrov/Dictionary.java

```diff
@@ -0,0 +1,56 @@
+import java.util.*;
+
+public class Dictionary {
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
+
+        List<String> bulgarianCities = new ArrayList<>();
+        bulgarianCities.add("Sofia");
+        bulgarianCities.add("Plovdiv");
+        bulgarianCities.add("Varna");
+
+        List<String> ukrainianCities = new ArrayList<>();
+        ukrainianCities.add("Kyiv");
+        ukrainianCities.add("Kharkiv");
+        ukrainianCities.add("Odesa");
+
+        Map<String,List<String>> dictionary = new HashMap<>();
+
+        dictionary.put("Bulgaria",bulgarianCities);
+        dictionary.put("Ukraine", ukrainianCities);
+
+        String input = scan.nextLine();
+
+        String result = "";
+
+        result = findCity(input, dictionary);
+
+        System.out.println(result);
+
+
+
+
+    }
+
+
+    public static String findCity(String input, Map<String, List<String>> countries){
+
+        String result = "No such country in database";
+
+        for (Map.Entry<String, List<String>> entry : countries.entrySet()){
+
+            for (String city: entry.getValue()){
+
+                if (city.equals(input)){
```

Comment:
Just a suggestion to use city.equalsIgnoreCase(input), because if you enter a lower case city it will return "No such country in database".